site_root/input/resources/profiles/StructureDefinition-kfdrc-research-study-no-phi.json

```diff
@@ -0,0 +1,33 @@
+{
+  "resourceType": "StructureDefinition",
+  "id": "kfdrc-research-study-no-phi",
+  "url": "http://fhir.kids-first.io/StructureDefinition/kfdrc-research-study-no-phi",
+  "version": "0.1.0",
+  "name": "kfdrc-research-study-no-phi",
+  "title": "Kids First DRC ResearchStudy without Protected Health Information",
+  "status": "draft",
+  "fhirVersion": "4.0.0",
+  "kind": "resource",
+  "abstract": false,
+  "type": "ResearchStudy",
+  "baseDefinition": "http://fhir.kids-first.io/StructureDefinition/kfdrc-research-study",
+  "derivation": "constraint",
+  "differential": {
+    "element": [
+      {
+        "id": "ResearchStudy",
+        "path": "ResearchStudy"
+      },
+      {
+        "id": "ResearchStudy.contact",
```

Comment:
I don't think this PHI since its in reference to the research study and not a patient

Reply:
Updated!